src/platform/selfsign_openssl.c

```diff
@@ -200,55 +256,113 @@ CxPlatTlsGenerateSelfSignedCert(
         goto Exit;
     }
 
-    Fd = fopen(PrivateKeyFileName, "wb");
+    if (!OutputPkcs12) {
+
+        Fd = fopen(PrivateKeyFileName, "wb");
+
+        if (Fd == NULL) {
+            QuicTraceEvent(
+                LibraryError,
+                "[ lib] ERROR, %s.",
+                "fopen() failed");
```

Comment:
nit: In all these "failed" logs please remove `()`.

Reply:
That was how it was when I moved the code from the original location.

Reply:
I'd still prefer it changed, since most all other code doesn't include it.